src/Application/Actions/Sitemap.php

```diff
@@ -40,61 +42,73 @@ protected function action(Request $request, Response $response, array $args): Re
-                limit: 100_000,
-                status: null,
-                metaCampaignSlug: null,
-                fundSlug: null,
-                jsonMatchInListConditions: [],
-                term:null,
-            );
 
-            foreach ($campaigns as $campaign) {
-                $endsInFuture = $campaign->getEndDate() > $this->now;
+        $xml = new SimpleXMLElement('<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"></urlset>');
+        $campaigns = $this->campaignRepository->search(
+            sortField: 'amountRaised',
+            sortDirection:  'asc',
+            offset: 0,
+            limit: 100_000,
+            status: null,
+            metaCampaignSlug: null,
+            fundSlug: null,
+            jsonMatchInListConditions: [],
+            term:null,
+        );
+
+        foreach ($campaigns as $campaign) {
+            $endsInFuture = $campaign->getEndDate() > $this->now;
+
+            $changeFreq = $endsInFuture ? 'daily' : 'monthly';
+            if ($campaign->isOpen($this->now)) {
+                $changeFreq = 'hourly';
+            }
 
-                $changeFreq = $endsInFuture ? 'daily' : 'monthly';
-                if ($campaign->isOpen($this->now)) {
-                    $changeFreq = 'hourly';
-                }
+            $this->addUrl(
+                xml: $xml,
+                url: $this->environment->publicDonateURLPrefix() . 'campaign/' . $campaign->getSalesforceId(),
+                changeFreq: $changeFreq,
+                priority: $endsInFuture ? '0.5' : '0.25',
+            );
 
+            if ($campaign->isOpen($this->now) && ! $campaign->isRegularGiving()) {
+                // regular giving donate pages are at a different address and in any case behind a login-wall,
+                // so not worth listing in sitemap.
                 $this->addUrl(
                     xml: $xml,
-                    url: $this->environment->publicDonateURLPrefix() . 'campaign/' . $campaign->getSalesforceId(),
-                    updatedAt: \DateTimeImmutable::createFromInterface($campaign->getUpdatedDate()),
+                    url: $this->environment->publicDonateURLPrefix() . 'donate/' . $campaign->getSalesforceId(),
                     changeFreq: $changeFreq,
-                    priority: $endsInFuture ? '0.5' : '0.25',
+                    priority: '0.5'
                 );
+            }
+        }
 
-                if ($campaign->isOpen($this->now) && ! $campaign->isRegularGiving()) {
-                    // regular giving donate pages are at a different address and in any case behind a login-wall,
-                    // so not worth listing in sitemap.
-                    $this->addUrl(
-                        xml: $xml,
-                        url: $this->environment->publicDonateURLPrefix() . 'donate/' . $campaign->getSalesforceId(),
-                        updatedAt: \DateTimeImmutable::createFromInterface($campaign->getUpdatedDate()),
```

Comment:
removed updatedAt since I realised the pages are actually updated much more often than this suggests, e.g. whenever the statistics based on donation activity change. At least for now (and probably forever) we can just leave that out of the sitemap and let the search engines figure out for themselves how often the pages change.